qiskit_optimization/runtime/qaoa_client.py

```diff
@@ -72,7 +79,28 @@ def __init__(
                 ansatz, the evaluated mean and the evaluated standard deviation.
             store_intermediate: Whether or not to store intermediate values of the optimization
                 steps. Per default False.
+            use_swap_strategies: A boolean on whether or not to use swap strategies when
+                transpiling. If this is False then the standard transpiler with the given
+                optimization level will run.
+            use_initial_mapping: A boolean flag that, if set to True (the default is False), runs
+                a heuristic algorithm to permute the Paulis in the cost operator to better fit the
+                coupling map and the swap strategy. This is only needed when the optimization
+                problem is sparse and when using swap strategies to transpile.
+            use_pulse_efficient: A boolean on whether or not to use a pulse-efficient transpilation.
+                If this flag is set to False by default. See https://arxiv.org/abs/2105.01063.
+            optimization_level: The transpiler optimization level to run if the swap strategies are
+                not used. This value defaults to 1 in the QAOA runtime.
+            rng_seed: An optional int to set the seed of the random generator that chooses the
+                initial point if no initial point is given.
```

Comment:
I undid this change here: fd01687

Reply:
I requested the change in this thread. https://github.com/Qiskit/qiskit-optimization/pull/288#discussion_r769758563

1. VQEClient does not set random values in the constructor even if initial_point is None. Why does `QAOAClient` need to set random numbers?
2. Using `np.random` affects users' environment. It's good to have a separate random number generator. But, I'm fine to remove `rng_seed`. In that case, I suggest using `default_rng()` instead of `np.random` to generate `initial_point`.

```python
import numpy as np
from qiskit_optimization.runtime import QAOAClient

np.random.seed(1)
a = np.random.rand(3)
print('w/o QAOAClient', a)

np.random.seed(1)
_ = QAOAClient()
a = np.random.rand(3)
print('w/ QAOAClient', a)
```

```
w/o QAOAClient [4.17022005e-01 7.20324493e-01 1.14374817e-04]
w/ QAOAClient [1.14374817e-04 3.02332573e-01 1.46755891e-01]
```

Reply:
I suggest the following change. https://github.com/Qiskit/qiskit-optimization/pull/288/files#r770274194